<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Drink(No excluisve)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="beerChekBox">
            <property name="text">
             <string>Beer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="juiceChekBox">
            <property name="text">
             <string>Juice</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="coffeeChekBox">
            <property name="text">
             <string>Coffee</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>OS(Exclusive)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="windowsChekBox">
            <property name="text">
             <string>Windows</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="macChekBox">
            <property name="text">
             <string>Mac</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="LinuxChekBox">
            <property name="text">
             <string>Linux</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>ChooseOne</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="AradioButton">
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="BRadioButton">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="CRadioButton">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="grabData">
       <property name="text">
        <string>grab data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setData">
       <property name="text">
        <string>set data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
